docker/test-server-mysql.yml

```diff
@@ -13,7 +13,7 @@ services:
         depends_on:
             mysql:
                 condition: service_healthy
-        pull_policy: always
+        pull_policy: if_not_present
```

Comment:
I have not tested this, but I am pretty sure that this would break test server deployments.
The tags of a PR do not change, so each time you deploy a PR to a test server, the same tag is used. This change would mean that newer versions of the image won't be pulled and therefore test server deployments would effectively be useless.

Reply:
Shouldn't it pull the image with the tag `ARTEMIS_DOCKER_TAG` that (I assume)  is set for the branch that is rolled out? But you're probably right, and this is not really a developer setup, I'll revert the change.

Reply:
The `ARTEMIS_DOCKER_TAG` for PRs is always `pr-<number>`. If you update a PR a new image replaces the previous image with this tag.